src/inc/quic_datapath.h

```diff
@@ -180,6 +181,8 @@ typedef struct CXPLAT_ROUTE {
     CXPLAT_ROUTE_STATE State;
     CXPLAT_RAW_TCP_STATE TcpState;
 
+    uint16_t DatapathType; // CXPLAT_DATAPATH_TYPE
```

Comment:
Why is this a `uint16_t` and not simply `CXPLAT_DATAPATH_TYPE`? And should it be _before_ the "copies memory up to this point" section?